components/ShadowLayer/examples/ShadowCornerRadiusExample.m

```diff
@@ -63,6 +63,8 @@ - (id)initWithFrame:(CGRect)frame {
     [sliderControl addTarget:self
                       action:@selector(sliderValueChanged:)
             forControlEvents:UIControlEventValueChanged];
+    sliderControl.accessibilityLabel = @"Change corner radius";
+    sliderControl.accessibilityHint = @"Updates the corner radius of the example view";
     [self addSubview:sliderControl];
```

Comment:
I've found for many of these slider examples that I also need to provide an accessibility value via the slider delegate because otherwise the slider reads out somewhat confusing (in the context of the examples) percentage values.

Reply:
Example: https://github.com/material-components/material-components-ios/pull/8942/files

Reply:
Done